StatBot/Cards/ActionButton.cs

```diff
@@ -4,91 +4,146 @@
 using StatBot.Interfaces;
 using StatBot.Models;
 using StatBot.Database.PostgresRepositories;
+using StatBot.Commands;
 namespace StatBot.Cards
 {
-    public class ActionButton : ICard
+    public class ActionButton
     {
-        public List<string> ButtonsName { get; set;  }
         public CardAction Action { get; set; }
+        public struct ButtonSettings
+        {
+            public int ActionType;
+            public int Statistic_Id;
+            public string ButtonTittle;
+            public bool IsStatistic;
+            public ButtonSettings(int Type, int Id, string Tittle, bool IsStat)
+            {
+                ActionType = Type;
+                Statistic_Id = Id;
+                ButtonTittle = Tittle;
+                IsStatistic = IsStat;
+            }
+        }
+        public ButtonSettings buttonSettings;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this field 'private' and encapsulate it in a 'public' property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS1104)